### PR TITLE
implement redis

### LINKED ISSUE #159: Redis setup

### PR DESCRIPTION
**Pull request by xuejinglu**
#### What's this PR do?

implement redis for events caching, prevent excessive api calls
#### Any background context you want to provide?

redis!
#### What are the relevant tickets/issues?

closes #159 
#### Where should the reviewer start?

server/placeSearchController/placeSearchController.js
#### Are there tests written yet? How should this be manually tested?

no
#### Questions:
- Does this add new dependencies?no
- Does our documentation need to be updated?no

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const app = express();
 // dotenv sets up the process.env variables. Put env variables in a .env file in your root
 require('dotenv').config({ path: `${__dirname}/../.env` });
 const PORT = process.env.PORT;
+const HOST = process.env.HOST;
```
Comment (abuddiga) [inline server/server.js]: What is HOST?

File: server/placeSearch/placeSearchController.js [hunk 2]
```diff
@@ -71,19 +100,25 @@ module.exports = {
...
-          const places = data.results;
-          const tagObj = {};
-          tagObj[options.tripsAppTag] = places.map(place => formatPlace(place, options));
-
-          return tagObj;
-        })
-        .catch(err => helpers.errorHandler(err, req, res, next))
-    ))
+    Promise.all(optionsArray.map(options => {
+      const key = '' + options.qs.query + options.qs.types; // eslint-disable-line
```
Comment (abuddiga) [inline server/placeSearch/placeSearchController.js]: Use ES6 syntax: `${options.qs.query}:${options.qs.types}`

Let's also separate query and types with a `:`

**Comment by abuddiga** [comment]
:red_circle: is
:tada: :family:

**Comment by boyajay** [comment]
:fork_and_knife: :cake:

**Comment by leranf** [comment]
so fresh and so clean cleannnn

**Pull request merged**